dotnet/src/Connectors/Connectors.Amazon/Bedrock/Core/Models/AI21Labs/AI21JurassicIOService.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using Amazon.BedrockRuntime.Model;
+using Microsoft.SemanticKernel.ChatCompletion;
+
+namespace Microsoft.SemanticKernel.Connectors.Amazon.Core;
+
+/// <summary>
+/// Input-output service for AI21 Labs Jurassic.
+/// </summary>
+internal sealed class AI21JurassicIOService : IBedrockModelIOService
+{
+    /// <summary>
+    /// Builds InvokeModelRequest Body parameter to be serialized.
+    /// </summary>
+    /// <param name="modelId">The model ID to be used as a request parameter.</param>
+    /// <param name="prompt">The input prompt for text generation.</param>
+    /// <param name="executionSettings">Optional prompt execution settings.</param>
+    /// <returns></returns>
+    object IBedrockModelIOService.GetInvokeModelRequestBody(string modelId, string prompt, PromptExecutionSettings? executionSettings)
```

Comment:
> [!IMPORTANT]
> Apply for all classes.

Please add all missing `access modifiers` for all methods

```suggestion
    object IBedrockModelIOService.GetInvokeModelRequestBody(string modelId, string prompt, PromptExecutionSettings? executionSettings)
```

Reply:
We came across [this issue](https://github.com/microsoft/semantic-kernel/pull/7587#discussion_r1706233531), when changing access modifiers. If the explicit implementation is removed, even with an internalized interface the functions cannot be marked as non-public

Reply:
Sure, I missed the issue, added a comment, suggest moving away from the explicit as this is more a workaround than actually intentional, lets keep those methods public but the class `internal`.